figure.html

```diff
@@ -286,8 +424,42 @@
 </li>
 
 <li>
+<p class="first">Click the <span class="bold">Info</span> tab and click on the <span class="bold">Set dpi</span> button to change the dpi settings  for a selected image.</p>
+
+<p>Interpolation will be used to achieve any non-default dpi setting.</p>
```

Comment:
I think this section might need a bit more elaboration for most users. E.g

Click the Info tab to see the resolution for selected panels in dpi. If you wish to *increase* the dpi of images within the exported PDF figure, you can choose a target resolution with the Set dpi button. Additional pixels will be created in the image and the data resampled using a bicubic filter. 

Also, it would be great to have the initial screen-shot with a low dpi, E.g. 50, since then it makes sense to boost it to 300. And if possible an "after" screenshot of the dpi display would be nice, since that shows the current and export dpi values together.